webapp/src/components/modals/create_issue/create_issue_form.tsx

```diff
@@ -62,9 +62,9 @@ const CreateIssueForm = ({theme, handleClose, isSubmitting, setIsSubmitting}: Pr
             title: issueTitle,
             description: issueDescription,
             project_id: project.project_id,
-            labels: labels.map((label) => label.value),
-            assignees: assignees.map((assignee) => assignee.value),
-            milestone: milestone?.value,
+            labels: labels.map((label) => label.value as string),
+            assignees: assignees.map((assignee) => assignee.value as number),
+            milestone: milestone?.value as number,
```

Comment:
Can we instead have these typed separately, rather than the `string | number | Issue` thing? Then we wouldn't need `as` here I think. It's fine this is the easiest way, just curious if it needs to be this way